lips/lip-6.mdx

```diff
@@ -0,0 +1,140 @@
+
+# Terminology
+* *Validator Owner* is an owner of the Libra account of the Libra Association's member. The account stores the *Validator Config* resource.
+* *Validator Config* is an on-chain resource that contains information for running a Validator Node. It includes a *consensus public key*, which is an EdDSA public key used to authenticate consensus messages. It also includes *network information*: network addresses and static public keys for establishing secure connections with this Validator Node from other Validator Nodes or from clients.
+* *Validator Set* is an on-chain resource that stores a vector, where each entry contains an address of a Validator Owner and a copy of its Validator Config. Though the Validator Config in the Validator Set may be an older version of the Validator Config currently stored under Validator Owner's account, each Validator Node is run with the Validator Config from the Validator Set.
+Each address appears at most once in the vector. The Libra Association collectively decides which Validator Owners' addresses are in the Validator Set. There is a one-to-one correspondence between a Validator Owner in the Validator Set and a Validator Node. The Validator Set changes, either when one of the Validator Nodes updates their parameters or when a new member is added or removed, the Reconfiguration happens.
+* *Validator Node* is a replica that runs the Libra Protocol to maintain the blockchain state. A Validator Node processes transactions and interacts with other Validator Nodes to reach consensus on the ordering of transactions and the resulting state of the database after transactions' executions.
+* *Validator Operator* is an owner of an on-chain Libra account, it may operate none, one or several Validator Nodes: a Validator Owner may designate a Validator Operator to run a Validator Node on Validator Owner's behalf. A Validator Operator may run multiple Validator Nodes. The Validator Operator for a Validator Owner has permission to rotate the Validator Config for the Validator Owner locally, as well as copy it to the Validator Set.
+* *Reconfiguration* defines a new epoch by triggerring a new epoch event (`LibraConfig::NewEpochEvent`), it gets emitted if any part of the global on-chain configuration, such as the Validator Set, changes. If a transaction causes an epoch change, that transaction is the last transaction committed in the current epoch — any subsequent transactions in that block or future blocks from that epoch will be ignored. Once the transaction has been committed, the new configuration starts being valid immediately. E.g. the new Validator Set will start running the consensus protocol in the new epoch.
+
+<img alt="Validator Set" src={useBaseUrl('img/validator_set.svg')} />
+
+# Specifications
+
+## Workflows
```

Comment:
I think I had some feedback here to break this up into two sections:
* a set of operations
* a set of workflows that give high level overview and dictate you through a series of operations

not sure the best ordering